Modals

datsenkoillia opened the pull request:
buy-modal and ingredients-modal

datsenkoillia committed:
buy now mpdal

datsenkoillia committed:
Chololate  icecream 320 4g  8,2g  40,4g  kcal Prote... Fats Carbs Ingredients Milk 100ml Belgian chocolate 20g Cream 150 ml Cocoa powder  30g Sugar 40g Vanilla sugar 10g Name Phone Comment Submit

datsenkoillia committed:
Update _modal.scss

RomarioNik commented on src/partials/modals/ingredients-modal.html:
```diff
@@ -1,7 +1,131 @@
+  </button>
+
+  <div class="modal-container">
+    <p class="modal__title ingredients-modal__title">Chololate icecream</p>
```
Может я ошибаюсь, но мне кажется, что слово "icecream" нужно писать раздельно "ice cream".
Думаю, что дизайнер не досмотрел, оно везде по макету отдельно

datsenkoillia replied:
В соответствии с макетом, не вижу смісла исправлять.

RomarioNik commented on src/partials/modals/ingredients-modal.html:
```diff
@@ -1,7 +1,131 @@
+      <ul class="ingredients-modal__kalories-list">
+        <li class="ingredients-modal__kalories-item">
+          <div class="ingredients-modal__kalories-description">kcal</div>
+          <div class="ingredients-modal__kalories-value">320</div>
```
36, 37 строки
Используй симантичные теги: для цифр можно <span>, для текста <p>
Ниже по коду так же, не буду коментить)

datsenkoillia replied:
для модалок семантику соблюдать необязательно вроде как.. 
исправлю на "p"

RomarioNik commented on src/partials/modals/ingredients-modal.html:
```diff
@@ -1,7 +1,131 @@
+          required
+          placeholder="Phone"
+          pattern="[\d+\-\/*%^()\s]+$"
+          title="Phone can notcontain letters."
```
"Блоху" нашел))
title="... not contain ..."

datsenkoillia replied:
Ок :)

RomarioNik reviewed (changes requested):
Модальні вікна для десктопної та планшетної версії однакові

datsenkoillia committed:
modals fix

- ширина для мобілки збільшена;
- змінено теги для тесту з div на p
- інші дрібні фікси

datsenkoillia committed:
Update _ingredients-modal.scss
